tests/ft_rust.vader

```diff
@@ -1,5 +1,11 @@
 Include: include/setup.vader
 
+Before (create src directory):
+  call mkdir('src')
+
+After (cleanup src directory):
+  !rmdir src
+
```

Comment:
Thanks, `make tests/ft_rust.vader` is much better :-)

But I still get those two kinds of errors.